torch/csrc/cuda/comm.cpp

```diff
@@ -86,10 +86,12 @@ std::vector<Tensor> broadcast(const Tensor& tensor, IntArrayRef devices) {
 // one of them is modified in-place during `forward` but the other is needed in
 // backward, autograd engine will complain.
 //
-// We thus re-wrap these Variables after broadcasting (i.e., effetively doing
+// We thus re-wrap these Variables after broadcasting (i.e., effectively doing
 // what is equivalent to .data in Python), and give them individual version
 // counters.
```

Comment:
Based on this comment, would it work if we just call `var.data()` instead of keeping the original `make_variable(var.tensor_data(), false)`?

Reply:
Maybe. I tried something similar and it didn't work; I'm going to have to study this site more carefully. Leaving for another patch.